tests/integration/upload_file/test_upload_file.py

```diff
@@ -0,0 +1,102 @@
+import base64
+import hashlib
+import os
+
+import yaml
+
+from tests.integration.base import DBTIntegrationTest, use_profile
+
+
+class TestUploadFile(DBTIntegrationTest):
+    local_file_path = './files/run/manifest.json'
```

Comment:
Does the test file need to be so big? I guess we can use just a small file like 1 line or so.

Reply:
True, replaced it with a small file instead!